src/main/java/uk/gov/companieshouse/api/accounts/model/rest/Costs.java

```diff
@@ -6,14 +6,14 @@
 public class Costs {
 
     @JsonProperty("costs")
-    private Map<String, Cost> costs;
+    private Map<String, Cost> costsList;
```

Comment:
Can this cause issue with spring-deserialising? usually the name of the field has to match the json property although i could be wrong (worth checking this)

Reply:
The point of JsonProperty is to rename a property.
https://github.com/FasterXML/jackson-annotations#annotations-for-renaming-properties

Reply:
yeah, investigated this and talked about it in slack, all good 👍